index.bs

```diff
@@ -1365,9 +1377,12 @@ and a [=boolean=] |topLevelFiltersMatch|, run the following steps:
 
 1. If |sourceToAttribute|'s [=attribution source/randomized response=] is not null and is not [=set/is empty|empty=]:
     1. Assert: |sourceToAttribute|'s [=attribution source/event-level attributable=] is false.
-    1. Return <strong>dropped</strong>.
-1. If |topLevelFiltersMatch| is false, return <strong>dropped</strong>.
-1. If |sourceToAttribute|'s [=attribution source/event report window time=] is less than the current time, return <strong>dropped</strong>.
+    1. Return [ "`status`" → "`dropped`" ].
+1. If |topLevelFiltersMatch| is false:
+    1. Let |debugData| be the result of running [=obtain debug data on trigger registration=]
+        with "`trigger-no-matching-filter-data`", |trigger| and |sourceToAttribute|.
+    1. Return [ "`status`" → "`dropped`",  "`debug_data`" → |debugData| ].
```

Comment:
Instead of returning a map here, should we introduce a struct type for this? It would allow fields to be cross-referenced automatically.

Reply:
If it's really verbose, you could use a tuple struct instead: https://infra.spec.whatwg.org/#tuples

Reply:
Updated with tuple, thanks.